### PR TITLE
src: update build command help text

### PR DESCRIPTION
**Pull request by lkingland**
- :broom:  Update build command help text a bit

### CODE REVIEW

**Review by lance — COMMENTED** [review]
Shouldn't this also include an update to the docs?

**Comment by lance** [comment]
/override "On Cluster RT Test (1.18.x, ubuntu-latest)"

**Comment by lkingland** [comment]
> Shouldn't this also include an update to the docs?

Oops! Yes of course.  It should be there now.

File: docs/reference/func_deploy.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ func deploy
 
 ```
       --build string[="true"]   Build the function. [auto|true|false]. [Env: $FUNC_BUILD] (default "auto")
-  -b, --builder string          builder to use when creating the underlying image. Currently supported builders are "pack" and "s2i". (default "pack")
+  -b, --builder string          builder to use when creating the underlying image. Currently supported builders are "pack" and "s2i". (default "s2i")
```
Comment (lance) [inline docs/reference/func_deploy.md:104]: This seems to have picked up a local change unnecessarily

Comment (lkingland) [reply docs/reference/func_deploy.md:104]: Ah yes, #1424 had not yet been merged!  Rebased...

**Pull request merged**